datalad/core/distributed/clone.py

```diff
@@ -652,6 +645,16 @@ def postclonecfg_annexdataset(ds, reckless, description=None):
             lgr.warning("reckless=ephemeral mode: Unable to create symlinks on "
                         "this file system.")
 
+    if reckless:
+        # we successfully dealt with reckless here.
+        # store the reckless setting in the dataset to make it
+        # known to later clones of subdatasets via get()
+        ds.config.set(
+            'datalad.clone.reckless', reckless,
+            where='local',
+            # delay reload until all config IO is done
+            reload=False)
```

Comment:
This lack of a reload is what's behind the new test failures.  In the deleted call above, the very next line has `reload=True`.

Reply:
Thx!